zerver/views/realm_filters.py

```diff
@@ -0,0 +1,35 @@
+from django.core.exceptions import ValidationError
+from django.http import HttpRequest, HttpResponse
+from django.views.decorators.csrf import csrf_exempt
+
+from zerver.models import realm_filters_for_domain, UserProfile
+from zerver.lib.actions import do_add_realm_filter, do_remove_realm_filter
+from zerver.lib.response import json_success, json_error
+from zerver.lib.rest import rest_dispatch as _rest_dispatch
+rest_dispatch = csrf_exempt((lambda request, *args, **kwargs: _rest_dispatch(request, globals(), *args, **kwargs)))
+
+
+# Custom realm filters
+def list_filters(request, user_profile):
+    # type: (HttpRequest, UserProfile) -> HttpResponse
+    filters = realm_filters_for_domain(user_profile.realm.domain)
+    return json_success({'filters': filters})
+
+def create_filter(request, user_profile):
```

Comment:
This (and the delete one) should use the `@require_realm_admin` decorator -- otherwise non-admins will be able to modify these.